other.go

```diff
@@ -0,0 +1,217 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/mattermost/mattermost-plugin-apps/apps/appclient"
+	"github.com/mattermost/mattermost-plugin-apps/utils"
+	"github.com/mattermost/mattermost-server/v6/model"
+
+	"github.com/mattermost/mattermost-app-test/path"
+)
+
+func otherCommandBinding(_ apps.Context) apps.Binding {
+	b := newBinding("dialog_from_interactive_message", path.CreateDialogMessage)
+	b.Submit = b.Submit.WithExpand(apps.Expand{ActingUserAccessToken: apps.ExpandAll})
+
+	return apps.Binding{
+		Label: "other",
+		Bindings: []apps.Binding{
+			b,
+		},
+	}
+}
+
+func initHTTPOther(r *mux.Router) {
```

Comment:
Is there a more meaningful name then `Other`? Maybe
```suggestion
func initHTTPDialog(r *mux.Router) {
```

Reply:
2/5 just drop this interactive dialog stuff. @larkox what was the value of the Interactive Dialog tests here?